.github/workflows/build-test.yml

```diff
@@ -29,8 +29,7 @@ jobs:
         with:
           path: ${{ steps.yarn-cache-dir.outputs.YARN_CACHE_DIR }}
           key: yarn-cache-${{ runner.os }}-${{ steps.yarn-version.outputs.YARN_VERSION }}-${{ hashFiles('yarn.lock') }}
-      - run: yarn --frozen-lockfile
-      - run: yarn allow-scripts
```

Comment:
This is not needed anymore, as with Yarn3 we can use the yarn plugin added in .yarn/plugins/@yarnpkg/plugin-allow-scripts.cjs